examples/dataflow-data-generator/data-generator-pipeline/tests/test_get_pyarrow_translated_schema.py

```diff
@@ -0,0 +1,77 @@
+import json
+import logging
+import pyarrow as pa
+import unittest
+from data_generator.ParquetUtil import get_pyarrow_translated_schema
+
+
+class TestPyarrowSchemaTranslator(unittest.TestCase):
+    def test_get_pyarrow_translated_schema(self):
```

Comment:
Once you add the functionality for RECORD type, change this test schema to have a nested field.
Also, once you add date/ timestamp handling similar to [`fix_record_for_avro`](https://github.com/GoogleCloudPlatform/professional-services/blob/master/examples/dataflow-data-generator/data-generator-pipeline/data_generator/AvroUtil.py#L17) you should construct a record and test the date handling similar to [`test_fix_record_for_avro`](https://github.com/GoogleCloudPlatform/professional-services/blob/master/examples/dataflow-data-generator/data-generator-pipeline/tests/test_fix_record_for_avro.py)